[2024-02-25T13:49:03.951+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: processing_sales.transform_data_to_silver scheduled__2022-09-20T01:00:00+00:00 [queued]>
[2024-02-25T13:49:03.960+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: processing_sales.transform_data_to_silver scheduled__2022-09-20T01:00:00+00:00 [queued]>
[2024-02-25T13:49:03.962+0000] {taskinstance.py:2171} INFO - Starting attempt 3 of 4
[2024-02-25T13:49:03.976+0000] {taskinstance.py:2192} INFO - Executing <Task(BigQueryInsertJobOperator): transform_data_to_silver> on 2022-09-20 01:00:00+00:00
[2024-02-25T13:49:03.981+0000] {standard_task_runner.py:60} INFO - Started process 2949 to run task
[2024-02-25T13:49:03.985+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'processing_sales', 'transform_data_to_silver', 'scheduled__2022-09-20T01:00:00+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/process_sales.py', '--cfg-path', '/tmp/tmp61k4514g']
[2024-02-25T13:49:03.987+0000] {standard_task_runner.py:88} INFO - Job 257: Subtask transform_data_to_silver
[2024-02-25T13:49:04.037+0000] {task_command.py:423} INFO - Running <TaskInstance: processing_sales.transform_data_to_silver scheduled__2022-09-20T01:00:00+00:00 [running]> on host 9e4f31030bba
[2024-02-25T13:49:04.116+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='processing_sales' AIRFLOW_CTX_TASK_ID='transform_data_to_silver' AIRFLOW_CTX_EXECUTION_DATE='2022-09-20T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-09-20T01:00:00+00:00'
[2024-02-25T13:49:04.127+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-02-25T13:49:04.132+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-02-25T13:49:04.133+0000] {bigquery.py:2806} INFO - Executing: {'query': {'query': "\n                    INSERT INTO `de-07-natalia-sokil.silver.sales` (client_id, purchase_date, product_name, price)\n                    SELECT\n                        CustomerId AS client_id,\n                        PARSE_DATE('%Y-%m-%d', PurchaseDate) AS purchase_date,\n                        Product AS product_name,\n                        CAST(REGEXP_REPLACE(Price, '[^0-9.]', '') AS FLOAT64) AS price\n                    FROM `de-07-natalia-sokil.bronze.sales`\n                ", 'useLegacySql': False}}'
[2024-02-25T13:49:04.177+0000] {bigquery.py:1596} INFO - Inserting job ***_processing_sales_transform_data_to_silver_2022_09_20T01_00_00_00_00_3f1ae23bae58be558431a81b4e9d4a99
[2024-02-25T13:49:05.690+0000] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 2864, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/query.py", line 1595, in result
    do_get_result()
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/query.py", line 1584, in do_get_result
    super(QueryJob, self).result(retry=retry, timeout=timeout)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/cloud/bigquery/job/base.py", line 952, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.BadRequest: 400 Mismatch between format character '-' and string character '/' at string index: 4 with format: '%Y-%m-%d'

Location: EU
Job ID: airflow_processing_sales_transform_data_to_silver_2022_09_20T01_00_00_00_00_3f1ae23bae58be558431a81b4e9d4a99

[2024-02-25T13:49:05.719+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=processing_sales, task_id=transform_data_to_silver, execution_date=20220920T010000, start_date=20240225T134903, end_date=20240225T134905
[2024-02-25T13:49:05.768+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 257 for task transform_data_to_silver (400 Mismatch between format character '-' and string character '/' at string index: 4 with format: '%Y-%m-%d'

Location: EU
Job ID: airflow_processing_sales_transform_data_to_silver_2022_09_20T01_00_00_00_00_3f1ae23bae58be558431a81b4e9d4a99
; 2949)
[2024-02-25T13:49:05.807+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-25T13:49:05.826+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
